spec/system/support/better_rails_system_specs.rb

```diff
@@ -17,10 +17,12 @@ def dom_id(*args)
   end
 end
 
+require "capybara-inline-screenshot/minitest"
 RSpec.configure do |config|
   # Add #dom_id support
   config.include ActionView::RecordIdentifier, type: :system
   config.include BetterRailsSystemTests, type: :system
+  config.include CapybaraInlineScreenshot::Minitest, type: :system
```

Comment:
would this work for the RSpec setup we have here? I'm exploring minitest as a default, but not there yet :)